opencti-platform/opencti-front/src/private/components/common/files/workbench/WorkbenchFileContent.jsx

```diff
@@ -243,6 +243,8 @@ const importValidation = (t) => Yup.object().shape({
   connector_id: Yup.string().trim().required(t('This field is required')),
 });
 
+const uniqStixDomainObjectsFields = ['name', 'type', 'pattern', 'identity_class', 'x_opencti_location_type'];
```

Comment:
If I understand well, you added the "pattern" field for indicator. Does "identity_class" & "x_opencti_location_type" are the only other field types possible in the platform ? (with name & type of course)

Reply:
I'm not sure if there are other types, I only gathered the ones I found in the code, they were already used, and I added pattern for indicators.